sdk/nodejs/types/input.ts

```diff
@@ -69993,7 +69993,7 @@ export namespace wafv2 {
         /**
          * The part of a web request that you want AWS WAF to inspect. See Field to Match below for details.
          */
-        fieldToMatch?: pulumi.Input<inputs.wafv2.RuleGroupRuleStatementByteMatchStatementFieldToMatch>;
+        fieldToMatch?: pulumi.Input<inputs.wafv2.RuleGroupRuleStatementXssMatchStatementFieldToMatch>;
```

Comment:
Is this just better shortening? It sounds like it should be `RuleGroupRuleStatementMatchStatementFieldToMatch`, since the rule statement appears in types nested from multiple properties of `RuleGroupRuleStatement`.

Reply:
Yeah ugh, shorter token heuristic picked this.  I'd like to try to give it a better name.